application/src/main/java/org/togetherjava/tjbot/commands/generic/DatabaseGetCommand.java

```diff
@@ -0,0 +1,86 @@
+package org.togetherjava.tjbot.commands.generic;
+
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.ChannelType;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.example.AbstractCommand;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.DatabaseException;
+import org.togetherjava.tjbot.db.generated.tables.Storage;
+import org.togetherjava.tjbot.db.generated.tables.records.StorageRecord;
+
+import java.util.Optional;
+
+/**
+ * Implementation of an example command to illustrate how to use a database.
+ * <p>
+ * The implemented command is {@code /dbput}. It acts like some sort of simple
+ * {@code Map<String, String>}, allowing the user to store and retrieve key-value pairs from the
+ * database.
+ * <p>
+ * For example:
+ *
+ * <pre>
+ * {@code
+ * !dbget hello
+ * // TJ-Bot: Saved message: Hello World!
+ * }
+ * </pre>
+ */
+public class DatabaseGetCommand extends AbstractCommand {
+    private static final Logger logger = LoggerFactory.getLogger(DatabaseGetCommand.class);
+    private final Database database;
+
+    /**
+     * Creates a new command listener, using the given database.
+     *
+     * @param database the database to store the key-value pairs in
+     *
+     *        If the message is in the wrong format, it will respond to the user instead of throwing
+     *        any exceptions.
+     */
+    public DatabaseGetCommand(Database database) {
+        super("dbget", "This command can set or retrieve values from a key-object database.", true);
+        this.database = database;
+    }
+
+    @Override
+    public void onSlashCommand(SlashCommandEvent event) {
+        if (event.getMember() == null || (event.getMember() != null
+                && !event.getMember().hasPermission(Permission.MESSAGE_MANAGE))) {
+            return;
+        }
+
+        String message = event.getCommandString();
```

Comment:
Still the incorrect way, like I said override the Command#getOptions method and make usage of that instead.
I really recommend _actually_ running commands, not testing.

This command doesn't even work.